sfcli/SmartFace.Cli/Infrastructure/ApiImplementation/CamerasRepository.cs

```diff
@@ -104,14 +106,21 @@ private CameraUpdateRequest UpdateCameraData(Camera originalCamera, CameraReques
                 TemplateGeneratorResourceId =
                     updateData.TemplateGeneratorResourceId ?? originalCamera.TemplateGeneratorResourceId,
                 FaceDetectorResourceId = updateData.FaceDetectorResourceId ?? originalCamera.FaceDetectorResourceId,
-                PedestrianDetectorResourceId = updateData.PedestrianDetectorResourceId ?? originalCamera.PedestrianDetectorResourceId
+                PedestrianDetectorResourceId = updateData.PedestrianDetectorResourceId ?? originalCamera.PedestrianDetectorResourceId,
+                SpoofDetectorConfig = originalCamera.SpoofDetectorConfig
             };
 
             updatedCamera.FaceDetectorConfig.MaxFaceSize = updateData.TrackMaxFaceSize ?? originalCamera.FaceDetectorConfig.MaxFaceSize;
             updatedCamera.FaceDetectorConfig.MinFaceSize = updateData.TrackMinFaceSize ?? originalCamera.FaceDetectorConfig.MinFaceSize;
-            
+            updatedCamera.FaceDetectorConfig.MaxFaces = originalCamera.FaceDetectorConfig.MaxFaces;
```

Comment:
Do we need these lines that just copy properties from original camera configs? We copy the whole configs in the initializer, so it should be handled

Reply:
Done, thanks, did not notice whole configs assignments